scipy/stats/tests/test_stats.py

```diff
@@ -2875,19 +2883,19 @@ def test_ttest_rel():
     y = (stats.norm.rvs(loc=5, scale=10, size=501) +
          stats.norm.rvs(scale=0.2, size=501))
     y[500] = np.nan
-    with warnings.catch_warnings():
-        warnings.filterwarnings('ignore', category=RuntimeWarning)
+
+    with np.errstate(invalid="ignore"):
```

Comment:
Had to sprinkle some of these, I am not quite sure why. Either, it is randomness, or more likely I guess my numpy version/system simply leads to slightly different errors (e.g. say `np.abs` maybe does not warn for me).